mix.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Erlcloud.Mixfile do
+use Mix.Project
+
+    def project do
+        [
+            app: :erlcloud,
+            version: "0.13.1",
```

Comment:
it's already 0.13.2 and I wonder if MIX support `git` based versioning alike rebar like [here](https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud.app.src#L6)